samples/dapr/README.md

```diff
@@ -21,40 +21,40 @@ This Radius application will have two [containers](https://docs.radapp.dev/conce
 
 The user-facing UI app (`frontend`) offers a portal for users to place orders. Upon creating an order, `frontend` uses [Dapr service invocation](https://docs.dapr.io/developing-applications/building-blocks/service-invocation/service-invocation-overview/) to send requests to `nodeapp`.
 
-The `frontend` container is configured with a [Dapr sidecar extension](https://docs.radapp.dev/reference/resource-schema/link-schema/dapr-schema/extension/) to add the sidecar container.
+The `frontend` container is configured with a [Dapr sidecar extension](https://docs.radapp.io/reference/resource-schema/dapr-schema/extension/) to add the sidecar container.
 
 <img src="images/frontend.png" alt="A diagram of the complete application" width=400 />
 
 ### `backend` container
 
 The order processing microservice (`backend`) accepts HTTP requests to create or display orders. It accepts HTTP requests on two endpoints: `GET /order` and `POST /neworder`.
 
-The `backend` container is configured with a [Dapr sidecar extension](https://docs.radapp.dev/reference/resource-schema/link-schema/dapr-schema/extension/) to add the sidecar container, along with a [Dapr Route](#routes) to model Dapr communication.
+The `backend` container is configured with a [Dapr sidecar extension](https://docs.radapp.io/reference/resource-schema/dapr-schema/extension/) to add the sidecar container, along with a [Dapr Route](#routes) to model Dapr communication.
 
 <img src="images/backend.png" alt="A diagram of the backend order processing service" width=600 />
 
 ## Routes
 
-Radius offers communication between services via [Routes](https://docs.radapp.dev/concepts/appmodel-concept/#routes).
+Radius offers communication between services via [Routes](https://docs.radapp.io/guides/author-apps/networking/overview/).
 
 ### Dapr service invocation
 
-In this quickstart we will be using a [Dapr HTTP invoke route](https://docs.radapp.dev/reference/resource-schema/link-schema/dapr-schema/http/) resource to model communication from `frontend` to `backend`. This allows `frontend` to use Dapr service invocation to interact with `backend`.
+In this quickstart we will be using a [Dapr HTTP invoke route](https://docs.radapp.io/reference/resource-schema/dapr-schema/) resource to model communication from `frontend` to `backend`. This allows `frontend` to use Dapr service invocation to interact with `backend`.
 
 <img src="images/invoke.png" alt="A diagram of the Dapr service invocation" width=500 />
```

Comment:
none of the images get rendered as there is no images folder. would be good to remove these code inserts